solution/backend/regulations/templates/regulations/partials/sidebar_right.html

```diff
@@ -1,23 +1,49 @@
 {% load string_formatters %}
 
 {{ categories|json_script:"categories" }}
+{{ node_list|json_script:"node_list" }}
```

Comment:
Safely output a Python object as JSON, wrapped in a <script> tag, ready for use with JavaScript.

https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#std-templatefilter-json_script